integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java

```diff
@@ -656,10 +649,8 @@ public int release(String path, FuseFileInfo fi) {
     LOG.trace("release({})", path);
     OpenFileEntry oe;
     final long fd = fi.fh.get();
-    synchronized (mOpenFiles) {
-      oe = mOpenFiles.getFirstByField(ID_INDEX, fd);
-      mOpenFiles.remove(oe);
-    }
+    oe = mOpenFiles.getFirstByField(ID_INDEX, fd);
```

Comment:
I think this now has different behavior without the lock?

Reply:
the different behavior is that `oe` might be removed between these two lines, but that's fine.